src/js/core/factories/GridColumn.js

```diff
@@ -464,8 +464,22 @@ angular.module('ui.grid')
       }
     }
 
-    self.minWidth = !colDef.minWidth ? 30 : colDef.minWidth;
-    self.maxWidth = !colDef.maxWidth ? 9000 : colDef.maxWidth;
+    ['minWidth', 'maxWidth'].forEach(function (name) {
```

Comment:
The `width` variable is already checked. 0_0
[https://github.com/merlinchen/ui-grid/blob/col-min-and-max-width-validity-check/src/js/core/factories/GridColumn.js#L435](https://github.com/merlinchen/ui-grid/blob/col-min-and-max-width-validity-check/src/js/core/factories/GridColumn.js#L435)